File: springboot/bean/src/test/java/com/spring/bean/beans/BeanConfigTest.java
package com.spring.bean.beans;

import com.spring.bean.domain.BeanDomainTest;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class BeanConfigTest {

    @Bean
    public BeanDomainTest beanMake() {
        return new BeanDomainTest();
    }

}

File: kotlin/study/src/main/kotlin/com/sample/study/annotation/bean/BeanConfigTest.kt
package com.sample.study.annotation.bean

import com.sample.study.generic.GenericTypeCheck
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.stereotype.Component

@Configuration
class BeanConfigTest {

    @Bean
    fun makeBeanTest(): String {
        println("[THIS IS MAKE BEAN TEST CHECK BEAN TEST METHOD !!!]")
        return "THIS IS BEAN CONFIG TEST MAKE BEAN METHOD !!"
    }

}

fun main() {

    GenericTypeCheck.specialHandlingByClass(test = "")

}